Description: Remove remove-dir-all utility
 We don't need this at the moment
Author: Zixing Liu <zixing.liu@canonical.com>
Forwarded: not-needed
Last-Update: 2023-09-19

Index: remove-dir-all/Cargo.toml
===================================================================
--- remove-dir-all.orig/Cargo.toml
+++ remove-dir-all/Cargo.toml
@@ -45,11 +45,6 @@ repository = "https://github.com/XAMPPRo
 name = "remove_dir_all"
 path = "src/lib.rs"
 
-[[bin]]
-name = "remove-dir-all"
-path = "src/bin/remove-dir-all.rs"
-required-features = ["cli"]
-
 [dependencies.cfg-if]
 version = "1.0.0"
 
@@ -92,12 +87,6 @@ version = "3.1"
 version = "0.2"
 
 [features]
-cli = [
-    "dep:clap",
-    "dep:env_logger",
-    "log",
-    "parallel",
-]
 default = []
 log = ["dep:log"]
 parallel = ["dep:rayon"]
